cronjobs/forwarder/index.ts

```diff
@@ -1,11 +1,17 @@
 import prisma from "@prismalocal/client";
 import { LightningAddressPaymentRequest } from "@library/zbd/requestInterfaces";
-import { payToLightningAddress } from "@library/zbd/zbdClient";
+import { ZBDGetChargeResponse } from "@library/zbd/responseInterfaces";
+import {
+  payToLightningAddress,
+  getPaymentStatus,
+} from "@library/zbd/zbdClient";
+import { PaymentStatus } from "@library/zbd/constants";
 const log = require("loglevel");
 log.setLevel(process.env.LOGLEVEL);
 
 const TIME_BETWEEN_REQUESTS = 2000; // 2 seconds
-const MIN_BATCH_FORWARD_AMOUNT = 10000; // min amount in msat to batch forward
+const MIN_BATCH_FORWARD_AMOUNT =
+  parseInt(process.env.MIN_BATCH_FORWARD_AMOUNT) ?? 100000; // min amount in msat to batch forward
```

Comment:
just double checked this and it'll evaluate to `NaN` in the event a non-coerceable string is run through `parseInt`
```suggestion
  parseInt(process.env.MIN_BATCH_FORWARD_AMOUNT) || 100000; // min amount in msat to batch forward
```